src/adb_tv.py

```diff
@@ -0,0 +1,109 @@
+"""
+This module provides utilities for interacting with Android TVs via ADB (Android Debug Bridge).
+It includes functions for managing ADB keys, connecting to devices, retrieving installed apps,
+and verifying device authorization.
+"""
+
+import asyncio
+import os
+from pathlib import Path
+from typing import Dict, Optional
+
+from adb_shell.adb_device_async import AdbDeviceTcpAsync
+from adb_shell.auth.keygen import keygen
+from adb_shell.auth.sign_pythonrsa import PythonRSASigner
+
+ADB_CERTS_DIR = Path(os.environ.get("UC_CONFIG_HOME", "./config")) / "certs"
```

Comment:
Please use the defined get_config_root() function in config.py to get the configuration path to avoid duplicating logic.